extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/NarayanaQuarkusConfiguration.java

```diff
@@ -0,0 +1,147 @@
+package io.quarkus.narayana.jta.runtime;
+
+import java.util.List;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class NarayanaQuarkusConfiguration {
+    // non final properties
+    /**
+     * config
+     */
+    @ConfigItem(name = "CoordinatorEnvironmentBean.commitOnePhase", defaultValue = "true")
+    public boolean commitOnePhase;
+    /**
+     * config
+     */
+    @ConfigItem(name = "ObjectStoreEnvironmentBean.objectStoreDir", defaultValue = "PutObjectStoreDirHere")
```

Comment:
The default value shouldn't be a random thing.

Reply:
Fair enough. We use that so that users are aware that they need to configure the location.